src/lib/authentication/utils.js

```diff
@@ -31,3 +31,7 @@ export const parseParams = (queryString = '') => {
   });
   return params;
 };
+
+export const parseParamsToString = (queryString = '') => {
+  return Qs.parse(queryString, { ignoreQueryPrefix: true });
```

Comment:
Totally agree with creating a reusable utility, but then can you please replace the code where it is used with this new function? e.g. just above, line 25
I am not sure we should have this utils in `authentication`, I guess it has nothing to do with that... I think it makes sense to move to somewhere more "global"?

Reply:
I deleted it since if we use the state instead of the URL params we don't need it anymore